### DISCUSSION TITLE
Pipeline after homepage banner

### DISCUSSION THREAD

**Discussion by vand3rlinden** [adityatelange/hugo-PaperMod#1451] [Q&A] [answered]
Hi, since a Hugo update on my Mac, I notice a "|" after my home page banner.
![SCR-20240306-opdh](https://github.com/adityatelange/hugo-PaperMod/assets/96340723/69ac61a8-43e0-464f-a501-56459ad66424)

I sometimes see this pipeline when other PaperMod users are using multi-language. However, since I do not use multi-language, the pipeline looks strange on my blog. 

How can I remove it?

Thanks in advance.

**Comment by adityatelange** [comment, marked as answer]
#1449
